Src/AutoMoq/AutoMockPropertiesCommand.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using Moq;
+using Ploeh.AutoFixture.Kernel;
+
+namespace Ploeh.AutoFixture.AutoMoq
+{
+    /// <summary>
+    /// A command that populates all public writable properties/fields of a mock object with anonymous values.
+    /// </summary>
+    public class AutoMockPropertiesCommand : ISpecimenCommand
```

Comment:
> It "unwraps" the mock's Object property

That it does, but it doesn't _use_ `mock` for anything other than calling `Execute` with `mock.Object`. Why does it need to do that 'unwrapping', then?

> and adds a stricter request specification to filter out dynamic proxy fields.

Couldn't that specification just be added to an `AutoPropertiesCommand` instance?

> IgnoreProxyMembersSpecification is not really intuitive, and is too specific to ever be reused (I think), so it's probably best to make it nested and private.

Couldn't you make it a private member of the `Customization` instead? Like [this](http://blog.ploeh.dk/2013/04/08/how-to-automatically-populate-properties-with-automoq)?